packages/storybook/stories/va-link.stories.jsx

```diff
@@ -0,0 +1,159 @@
+import React, { Fragment } from 'react';
+import { getWebComponentDocs, propStructure, StoryDocs } from './wc-helpers';
+
+const linkDocs = getWebComponentDocs('va-link');
+
+export default {
+  title: 'Components/va-link',
+  parameters: {
+    componentSubtitle: `Link web component`,
+    docs: {
+      page: () => <StoryDocs data={linkDocs} />,
+    },
+  },
+};
+
+const defaultArgs = {
+  active: undefined,
+  channel: undefined,
+  download: undefined,
+  href: 'https://www.va.gov',
+  filename: undefined,
+  video: undefined,
+};
+
+const Template = ({ active, channel, download, href, filename, video }) => {
+  return (
+    <va-link
+      active={active}
+      channel={channel}
+      download={download}
+      href={href}
+      filename={filename}
+      video={video}
+    >
+      Find out if you qualify for this program and how to apply
+    </va-link>
+  );
+};
+
+export const Default = Template.bind(null);
+Default.args = {
+  ...defaultArgs,
+};
+Default.argTypes = propStructure(linkDocs);
+
+const ActiveTemplate = ({
+  active,
+  channel,
+  download,
+  href,
+  filename,
+  video,
+}) => {
+  return (
+    <va-link
+      active={active}
+      channel={channel}
+      download={download}
+      href={href}
+      filename={filename}
+      video={video}
+    >
+      Share your VA medical records
+    </va-link>
+  );
+};
+
+export const Active = ActiveTemplate.bind(null);
+Active.args = {
+  ...defaultArgs,
+  active: true,
+};
+
+const DownloadTemplate = ({
+  active,
+  channel,
+  download,
+  href,
+  filename,
+  video,
+}) => {
+  return (
+    <va-link
+      active={active}
+      channel={channel}
+      download={download}
+      href={href}
+      filename={filename}
+      video={video}
+    >
+      Download VA form 10-10EZ
+      <dfn>
+        (<abbr title="Portable Document Format">PDF</abbr>, 5 pages)
```

Comment:
Good catch. I've been unable to get the `abbr` element to use the correct color based on the status of the anchor. It seems I am not able to rely on the `:visited` selector to update the color of the nested `abbr`. Any ideas @bkjohnson ?

This works:
```css
va-link abbr {
  color: var(--color-link-default) !important;
}
```

This doesn't work:
```css
va-link a:visited abbr {
  color: var(--color-link-visited) !important;
}
```

```css
a:visited ::slotted(*) {
  color: var(--color-link-visited) !important;
}
```

Help! 🆘

Reply:
As a workaround, we could do something like this:

```diff
diff --git a/packages/web-components/src/components/va-link/va-link.tsx b/packages/web-components/src/components/va-link/va-link.tsx
index 3c19d8e5..2cb1f14a 100644
--- a/packages/web-components/src/components/va-link/va-link.tsx
+++ b/packages/web-components/src/components/va-link/va-link.tsx
@@ -44,6 +44,16 @@ export class VaLink {
    */
   @Prop() filename?: string;
 
+  /**
+   * The type of the file. Only valid if download is `true`.
+   */
+  @Prop() filetype?: 'PDF';
+
+  /**
+   * The number of pages of the file. Only valid if download is `true`.
+   */
+  @Prop() pages?: number;
+
   // @Prop() type?: string;
 
   /**
@@ -74,15 +84,23 @@ export class VaLink {
     });
   };
 
+  private getAbbreviationTitle = () => {
+    const { filetype } = this;
+    if (!filetype) return;
+    if (filetype === 'PDF') return 'Portable Document Format';
+  };
+
   render() {
     const {
       active,
       channel,
       download,
+      filetype,
       filename,
+      getAbbreviationTitle,
       handleClick,
       href,
-      // type
+      pages,
       video,
     } = this;
 
@@ -91,8 +109,6 @@ export class VaLink {
         <a
           href={href}
           download={download ? filename : undefined}
-          // Property 'type' does not exist on type 'AnchorHTMLAttributes<HTMLAnchorElement>'
-          // type={download ? type : undefined}
           onClick={handleClick}
           rel={channel || video ? 'noopener' : undefined}
           target={channel || video ? '_blank' : undefined}
@@ -108,6 +124,12 @@ export class VaLink {
               <slot></slot>
             </Fragment>
           )}
+          {download && filetype && (
+            <dfn>
+              (<abbr title={getAbbreviationTitle()}>{filetype}</abbr>
+              {pages && { pages }})
+            </dfn>
+          )}
         </a>
       </Host>
     );
```

Reply:
This looks like a Chrome bug, because in firefox it looks like how I expect it to:

!["PDF" abbreviation is the same color as the surrounding text](https://user-images.githubusercontent.com/2008881/178779016-07e74bb2-53cd-435b-b786-2f31e4e8cb3a.png)

However, since the problem still exists in Chrome we aren't going to be able to solve it with just CSS due to the `<abbr>` being nested inside of another element in the slot. Solutions I can think of are:

1. Move the `(<abbr title="Portable Document File">PDF</abbr>), N pages)` inside of the component
    - We could give a prop for the page count, or parse the pdf with [a dependency like this](https://github.com/adrienjoly/npm-pdfreader) to get the count ourselves. I saw your earlier comment about not wanting to have a user's browser download a pdf just to show the page count though, and I kind of agree with that.
1. Use some JS to add a class to the nested `<abbr>` to achieve the desired styling like we're doing elsewhere:
    - https://github.com/department-of-veterans-affairs/component-library/blob/45608acf85449782307945c77e56994a3ceaacb8/packages/web-components/src/components/va-modal/va-modal.tsx#L292

The simplest solution may be the extra prop.

Reply:
I agree about having the slot or the props, but not both.

Reply:
Agreed, I will remove the slot to make it easier to use and bake in the `<dfn>` and `<abbr>` elements.